node/src/main/resources/defaults.conf

```diff
@@ -331,3 +331,10 @@ metrics {
   zipkin = false
   sigar = false
 }
+
+dev-mode = false
+
+dev {
+  # If set, on each propose node will add dummy deploy signed by this key.
+  # deployer-private-key =
+}
```

Comment:
Add new line at the end.

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-800021338